### ISSUE TITLE
Using any theme makes addon-a11y crash

### ISSUE THREAD

**Issue by Hypnosphi** [storybooks/storybook#5780]
1. Use a theme, e.g.:

```
setOptions({ theme: themes.dark })
```

2. Open accessibility tab

Page crashes with
```
Uncaught TypeError: Cannot read property 'name' of undefined
```

**Comment by ndelangen** [comment]
@Hypnosphi Setting a theme as a parameter is recommended:

```jsx
// config.js

import { themes } from '@storybook/theming';
import { configure, addParameters } from '@storybook/react';

addParameters({
  options: {
    theme: light,
  },
});
```

That should work, but I'll look into why the code you posted results in an error.

**Comment by shilman** [comment]
@ndelangen I repro'd with parameter style in official-storybook, FYI

**Comment by ndelangen** [comment]
I see, will investigate

**Comment by ndelangen** [comment]
Fixed in https://github.com/storybooks/storybook/pull/5787

**Comment by shilman** [comment]
@ndelangen Still seeing this on `rc.9` 😦

**Comment by Hypnosphi** [comment]
Same on `rc.10`. I think I'll disable a11y addon for now

**Comment by shilman** [comment]
Yippee!! I just released https://github.com/storybooks/storybook/releases/tag/v5.0.0-rc.11 containing PR #5843 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
